<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetVerticalLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetVerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="acq_gbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="num_triggers_ledit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="num_images_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="energy_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Energy (keV):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="transmission_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Transmission (%):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="energy_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="exp_time_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Exposure time (s):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="num_images_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Number of images:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="num_images_per_trigger_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="exp_time_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="resolution_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="transmission_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="images_per_trigger_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1200</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Images per trigger:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="resolution_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Resolution (Ã…):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="num_triggers_label">
          <property name="text">
           <string>Number of triggers:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="detector_roi_mode_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1200</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Detector mode:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QComboBox" name="detector_roi_mode_combo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Select necesaary detector mode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="exp_time_total_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="total_exp_time_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Total exp. time (s):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="hare_num_label">
          <property name="text">
           <string>HARE number:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="hare_num_ledit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
